packages/config/src/functions_config.js

```diff
@@ -10,8 +10,11 @@ const WILDCARD_ALL = '*'
 // Removing the legacy `functions` from the `build` block.
 // Looking for a default directory in the `functions` block, separating it
 // from the rest of the configuration if it exists.
-const normalizeFunctionsProps = function ({ functions: v1FunctionsDirectory, ...build }, functions) {
-  const functionsA = Object.entries(functions).reduce(normalizeFunctionsProp, {})
+const normalizeFunctionsProps = function (
+  { functions: v1FunctionsDirectory, ...build },
+  { [WILDCARD_ALL]: wildcardProps, ...functions },
```

Comment:
This is defaulted to an empty object right before this function is called, so it always exists.